src/main/java/org/candlepin/insights/orgsync/db/DatabaseOrgListStrategy.java

```diff
@@ -20,25 +20,24 @@
  */
 package org.candlepin.insights.orgsync.db;
 
-import org.candlepin.insights.orgsync.OrgListStrategy;
+import org.candlepin.insights.orgsync.db.model.OrgConfig;
 
-import java.util.List;
+import java.util.stream.Stream;
 
 /**
  * Pulls the list of orgs to sync from a database table.
  *
- * See {@link Organization}.
+ * See {@link OrgConfig}.
  */
-public class DatabaseOrgListStrategy implements OrgListStrategy {
+public class DatabaseOrgListStrategy {
```

Comment:
Since we removed the 'strategy' concept, let's rename this to just DatabaseOrgList or something similar.

Reply:
@mstead, done. Let me know if you want me to squash.